core/shared/src/main/scala/zio/Fiber.scala

```diff
@@ -407,7 +407,7 @@ object Fiber {
    * @param id            The fiber's unique identifier
    * @param interruptors  The set of fibers attempting to interrupt the fiber or its ancestors.
    * @param executor      The [[zio.internal.Executor]] executing this fiber
-   * @param children      The fiber's forked children. This will only be populated if the fiber is supervised (via [[ZIO#supervised]]).
```

Comment:
We should keep this documentation even if we have to take out the link.

Reply:
If I understand correctly, `Fiber`s are always supervised now, no?

Reply:
Great point!